lw1551/Power of Thor intro.cs

```diff
@@ -0,0 +1,86 @@
+// *************
+// Power of Thor
+// *************
+// This is my solution for the `Power of Thor <https://www.codingame.com/training/easy/power-of-thor-episode-1/solution>`_
+//
+//
+// =====================
+// Thought Process
+// =====================
+// * Thor needs to be able to move based on the different locations of the power orb.
+// * Thor currently just walks off the screen and goes out of bounds due to being stuck in a while loop that does not check to see thor's location in relation to either the light or the border of the screen.
+// * There is going to need to be a way to increment Thor’s location in at least two variables depending on the cardinal direction he moves in.
+// * Make a variable that is Thor’s updated location after he moves from his initial location, then change this variable based on checking where thor is in relation to the power orb.
+// * Use *if statements* to check for thor’s updated location compared to the orb, then use *else statements* to compare them for each direction that thor can move in.
+// 
+// .. image:: Power_Of_Thor.png
+//
+// The idea is that Thor has an x and y position that is separate from the x and y position of the power, and this image shows that the plane is 40 wide by 18 high, but due to arrays this runs from 0 to 39 and 0 to 17.
+//
+// -------------------
+// Desgin Constraints
+// -------------------
+// 1. Thor cannot move beyond the coordinate plane defined by the game, which is **40 width by 18 height**.
+//
+// 2. Thor cannot run out of energy; the game allows for a **maximum of 100 energy**.
+//
+// ---------------------
+// Defined Variables
+// ---------------------
+//
+        string[] inputs = Console.ReadLine().Split(' ');
+        // The X position of the light of power
+        int lightX = int.Parse(inputs[0]); 
+        // The Y position of the light of power
+        int lightY = int.Parse(inputs[1]); 
+        // Thor's starting X position
+        int initialTX = int.Parse(inputs[2]); 
+        // Thor's starting Y position
+        int initialTY = int.Parse(inputs[3]); 
+        // The first two variables, ``lightX`` and ``lightY``, are used to set where the coordinates of the light orb's location for each new level of the game. I then defined ``initialTX`` and ``initialTY`` to be Thor's initial location. When first designing this program, I had two addtional variables that would update to thor's new location, but I realized when working through this that the ``initialTX`` and ``initialTY`` variables can be used for both functions.
+        string directionX = "";
```

Comment:
These variables make no sense here, since you re-define them a few lines later. Remove.

Reply:
I have removed these lines and updated comments for clarity purposes.